### PR TITLE
OPSEXP-2542 Fix build of arm64 RPMs

### PR DESCRIPTION
**Pull request by gionn**
* Do not build with libgs also for RPM
* Use arm64 runners instead of cross-compiling
* Reuse tests script in both build and test phase

ref OPSEXP-2542

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -142,7 +150,7 @@ jobs:
           file: ImageMagick-${{ steps.variables.outputs.imagemagick_version }}.${{ matrix.target_arch }}.rpm
           classifier: ${{ matrix.nexus_classifier }}
           files: ImageMagick-libs-${{ steps.variables.outputs.imagemagick_version }}.${{ matrix.target_arch }}.rpm
-          classifiers: ${{ matrix.nexus_classifier }}-libs
+          classifiers: libs-${{ matrix.nexus_classifier }}
```
Comment (gionn) [inline .github/workflows/build.yml:153]: this was making insane to manage within ansible:
https://github.com/Alfresco/alfresco-ansible-deployment/pull/892/files#diff-ae5275442eb2794776bfda0cabc6599958cef4a0c8100cec39ef7be7adc9dd29R13

**Review by pmacius — DISMISSED** [review]
Looks good to me

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (gionn) [reply .github/workflows/build.yml:153]: we are using the same imagemagick version 7.1.0-16 across all the supported acs versions

the new playbook will requires this newly built packages while the previous playbook versions would still use the previous ones

Comment (alxgomz) [reply .github/workflows/build.yml:153]: OK so just one new package for each distro. Make sense.

**Review by alxgomz — APPROVED** [review]

**Review by pmacius — APPROVED** [review]

**Pull request merged**